app/models/changeset/pull_request.rb

```diff
@@ -40,11 +40,20 @@ def self.changeset_from_webhook(project, params = {})
 
   def self.valid_webhook?(params)
     data = params['pull_request'] || {}
-    return false unless data['state'] == 'open'
+    return false unless data['state'] == 'open' && valid_action?(params['github']['action'])
+
+    if params['github']['action'] == 'edited'
```

Comment:
`action = params['github']['action']` to reuse it ... maybe even 
`params.require(:github).require(:action)` for nicer errors